src/inmanta/agent/config.py

```diff
@@ -188,7 +189,7 @@ def is_executor_mode(value: str | AgentExecutorMode) -> AgentExecutorMode:
     3,
     "Maximum number of concurrent executors to keep per environment, per agent. If this limit is already reached "
     "when creating a new executor, the oldest one will be stopped first.",
-    is_int,
+    functools.partial(is_lower_bounded_int, 1),
```

Comment:
We use the docstring of the function to generate documentation, so this won't look so pretty in the docs